c2corg_ui/static/js/auth/authservice.js

```diff
@@ -131,22 +130,16 @@ app.Authentication.prototype.hasEditRights = function(doctype, options) {
 
 /**
  * Checks if the current user has rights to access/edit the outing.
- * @param {!Array<string> | !string} users
+ * @param {string} users
```

Comment:
This type seems incorrect.
The first line of the function is
```
var u = typeof users === 'string' ? JSON.parse(users) : users;
```
=> this means that ``users`` is not always a string. It seems according to 
```
for (var i = 0; i < u.length; i++) {
```
that it may be an array.

``@param {string|Array.<string>} users`` seems to pass with Closure.